content/06.model-complexity.md

```diff
@@ -11,30 +11,31 @@ All of these contribute to low signal to noise ratio in rare disease datasets.
+Combining these three methods resulted in better overall prediction when implemented on a silver standard dataset, as compared to a model that used ensemble-based prediction alone. [TODO: my understanding of the baseline model is that there was no feature selection or prediction refinement, but that prediction refinement did not get tested alone as the earlier language implied -- check my understanding]
+In addition to cascade learning, other approaches that can better represent rare classes like inverse class weighting [TODO: Add ref?] and oversampling [doi:10.1613/jair.953] may also help mitigate limitations due to class imbalance. 
 
 The presence of label-noise and sparsity in the data can lead to overfitting of models to the training data, meaning that the models show high prediction accuracy on the training data but low prediction accuracy (and large prediction errors) on new evaluation data. 
-Overfit models tend to rely on patterns that are unique to the training data (for example, the calibration of the instrument that was used to generate the training data), and not generalizable to new data (e.g., data generated on the same instrument that has been recalibrated). [@doi:10.1073/pnas.1900654116]
-In such cases, regularization can not only protect ML models as well as learned representations from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. (Figure[@fig:3]a)
-Regularization is an approach by which a penalty is added to the model to avoid making large prediction errors. 
-Some examples of ML methods with regularization include ridge regression, LASSO regression, and elastic net regression, among others.
+Overfit models tend to rely on patterns that are unique to the training data (for example, the calibration of the instrument that was used to generate the training data), and not generalizable to new data (e.g., data generated on the same instrument that has been recalibrated). [@doi:10.1073/pnas.1900654116] [TODO: ref is about interpretability, little mention of generalizability outside discussion of use of accuracy?]
+Regularization can help in these scenarios.
+Regularization is an approach by which a penalty or constraint is added to a model to avoid making large prediction errors.
+These procedures can not only protect ML models as well as learned representations from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. (Figure[@fig:3]a)
+Some examples of ML methods with regularization include ridge regression, LASSO regression, and elastic net regression [@doi:10.1111/j.1467-9868.2005.00503.x], among others.
```

Comment:
A little bit of a reorganization so we define regularization before we talk about what it can _specifically_ help with.

Reply:
Looks great! 27 and 28 could possibly be combined ,but I don't think it's that important.